lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -582,6 +615,9 @@ export class ResponseHandler {
             extExpiresOn = new Date(
                 Number(cacheRecord.accessToken.extendedExpiresOn) * 1000
             );
+            refreshOn = new Date(
+                Number(cacheRecord.accessToken.refreshOn) * 1000
```

Comment:
What happens if "refreshOn" is null? This is quite possible: 

- ESTS doesn't emit refresh_in at all today
- ESTS will only emit refresh_in in certain cases anyway

Reply:
(this is a blocking comment)